src/lib.rs

```diff
@@ -1,31 +1,29 @@
-extern crate regex;
-extern crate crypto;
-
-use std::i64;
-use regex::Regex;
-use self::crypto::digest::Digest;
-use self::crypto::sha3::Sha3;
+use crypto::{digest::Digest, sha3::Sha3};
 
 pub fn checksum(address: &str) -> String {
-    let re = Regex::new(r"^0x").unwrap();
-    let address = address.to_lowercase();
-    let address = re.replace_all(&address, "").to_string();
+    let address = address.trim_start_matches("0x").to_lowercase();
```

Comment:
This is not ideal because it will remove the prefix multiple times, but the other function is nightly atm:
https://doc.rust-lang.org/std/primitive.str.html#method.strip_prefix

Reply:
See my other comment, we could improve it by having a more diverse API depending on the needs.
WE usually use `[u8, 20]` and we don't even need string, since we deserialize it directly (prefixed/non-prefixed)